src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -46,7 +58,9 @@ import { handleUnknownError } from '../../../utils/helper';
 import QrScan from '../../../../icons/qr-scan.svg?vue-component';
 
 export default {
-  components: { Modal, Button, QrScan },
+  components: {
+    Modal, Button, QrScan,
```

Comment:
I suggest to split the component entries into separate lines to prevent GIT from marking this line as changed in case new component is added.

Reply:
It will be marked anyway, because new component will be added there

Reply:
This is rather subjective, but in practice imagine a situation that 2 of the developers add 2 different components which is a common situation. It's harder to resolve the conflict as this will be one line. Having those in separate lines it's easier to resolve because we are picking the correct line/-s.